src/patternfly/components/ClipboardCopy/clipboard-copy-expanded-content.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="pf-c-clipboard-copy__expanded-content{{#if clipboard-copy-expanded-content--modifier}} {{clipboard-copy-expanded-content--modifier}}{{/if}}"
```

Comment:
can you change this to `pf-c-clipboard-copy__content` to match the expandable component, or maybe `pf-c-clipboard-copy__expandable-content` to match table/data-list?

Reply:
going to go with the second